tests/Baseline/spicy.types.vector.legacy-syntax-fail/output

```diff
@@ -1,6 +1,10 @@
 ### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
 [error] <...>/legacy-syntax-fail.spicy:7:8-7:20: vector<T> syntax is no longer supported for parsing sequences; use T[] instead.
+[error] invalid attribute '&length'
```

Comment:
well less so since we had to add back validation in the lexer :( hopefully that can be resolved another way so that attributes can be validated. The reason this error will be staying is that the lexer allows it but `Attribute` doesn't have a `Kind` for it - which is probably how deprecated attributes should act.

Unfortunately the error for trying to use an attribute that doesn't exist like `: uint8 &omg` is really unhelpful still, since it doesn't even count it as an attribute:

```
[error] test.spicy:4:13-4:13: syntax error, unexpected '&'
```

So a minor win now (it's nice that we can accept some attributes in the lexer then reject it with a nicer error if we know about them). It could be a bigger win if we find a better way to solve the attribute vs. bitwise `&` problem.